lib/util.js

```diff
@@ -62,17 +62,26 @@ exports.deprecate = function(fn, msg) {
   }
 
   var warned = false;
+  const warnedLines = new Set();
+
   function deprecated() {
+    var location = (new Error()).stack.match(/.*?\n.*\n.*?\((.*?:\d+):/)[1];
```

Comment:
This doesn't seem to match the correct line of the call. Here's a random test I put into an app of mine:

Stack:
````
Error
    at startListeners (/Users/silverwind/git/droppy/server/server.js:121:18)
    at /Users/silverwind/git/droppy/server/server.js:77:45
    at /Users/silverwind/git/droppy/node_modules/async/lib/async.js:695:13
    at iterate (/Users/silverwind/git/droppy/node_modules/async/lib/async.js:256:13)
    at doNTCallback0 (node.js:408:9)
    at process._tickCallback (node.js:337:13)
````
Output of your regex:
````
/Users/silverwind/git/droppy/node_modules/async/lib/async.js:256
````

You should try matching the first line here (server.js:121).